libs/RobotKit/include/StateMachine.h

```diff
@@ -101,7 +101,8 @@ namespace sm::action {
 	};
 
 	struct start_sleep_timeout {
-		auto operator()(irc &rc) const { rc.startSleepTimeout(); }
+		int duration = 60;
```

Comment:
Do we have to specify a value here ?

Reply:
If not specified, the default value is 0, so instead of having an instant timeout there is a default one to 60s, just in case